### ISSUE TITLE
The difference between strong_baseline and MMT proposed in unsupervised domain adaptation

### ISSUE THREAD

**Issue by GJTNB** [open-mmlab/OpenUnReID#22]
I see that this benchmark network is from sec3.1 in the MMT paper. How is it trained? Does it require collaborative training?

**Comment by yxgeee** [comment]
It does not require collaborative training. Only one network is used for training, supervised by a cross-entropy loss and a triplet loss.

**Comment by GJTNB** [comment]
Do you use clustering? Are there any techniques in clustering? Or is this benchmark based on that paper?

**Comment by yxgeee** [comment]
Yes, clustering is used for generating pseudo labels. Please refer to the code for the details. If you understand Chinese, you could also refer to https://apposcmf8kb5033.h5.xeknow.com/st/9k0kMekYC, where I have mentioned the architectures of this codebase, as well as the strong baseline design.
